crypten/mpc/primitives/arithmetic.py

```diff
@@ -140,7 +140,7 @@ def share(self, value):
         self._tensor = value
 
     @staticmethod
-    def from_shares(share, precision=None, src=0, device=None):
+    def from_shares(share, precision=None, device=None):
```

Comment:
Getting failure in CirclCI tests:

```
Traceback (most recent call last):
  File "/home/circleci/CrypTen/test/multiprocess_test_case.py", line 199, in _run
    getattr(self, test_name)()
  File "/home/circleci/CrypTen/test/test_mpc.py", line 117, in test_from_shares
    encrypted_tensor = MPCTensor.from_shares(share, ptype=ptype)
  File "/home/circleci/CrypTen/crypten/mpc/mpc.py", line 163, in from_shares
    result._tensor = from_shares(share, precision=precision, src=src)
TypeError: from_shares() got an unexpected keyword argument 'src'
```

This is caused by removing the `src` kwarg, which I think makes sense to do. However we need to adjust this in `mpc.py` line 163.